ext/standard/hrtime.h

```diff
@@ -1,56 +0,0 @@
-/*
```

Comment:
It's not absolutely necessary, but it felt right to not have Zend/ depend on ext/ (for no particular reason other than Zend/ being a lower layer than ext/)

Reply:
Moving code has a few disadvantages:

* It makes reviewing harder, because it's difficult to determine what code is new and what code was just moved. Sometimes small changes can sneak into moved code too, which is very hard to catch without using a diff tool.
* It creates merge conflicts when merging changes to the moved code. This is more of a problem for us than a "standard" code base because we have 3-4 active branches.
* It breaks git blame.

Of course, not refactoring also has downsides, e.g. leading to big, messy files, or making things hard to find.

I don't mind either way, you can decide what you see fit.

Reply:
I agree with your points. In this case I will retain the relocation of the code because, since I've also moved most of it from the .c to the .h for inlining purposes, keeping it in ext/ would not cancel all of these disadvantages.